### DISCUSSION TITLE
Simple Sample Sending Emails using Blazor 6

### DISCUSSION THREAD

**Discussion by greerda** [quartznet/quartznet#2311] [General]
I am confused.  I have a . NET 6 blazor server application that needs to send out emails on a specific date chosen by the user.  How do I do this?  Can someone send me the K(eep) I(t) S(imple) S(tupid) sample and instructions?

**Comment by jafin** [comment]
You should be able to follow examples for asp.net, Blazor is no different.
There should be examples in repo for syntax, in general something like should work:

- Register Quartz in startup via DI
- After your app captures when a user wants to send email, create a Trigger for the specified time, add to the quartz scheduler
- Create a job that as part of its execution sends the email with the appropriate context


Alternatively, if you have a lot of users but jobs still occur daily you could:
- Register Quartz in startup via DI
- Create a trigger to run daily, 
- create a job that queries your db for the list of emails that need to be dispatched, iterate through each item that needs dispatching and send emails from within the job.
- If there are no users who need to send emails for that period it is a noop.
